markdown/ch7.markdown

```diff
@@ -95,11 +96,32 @@ whether jiggling it in one direction or another reduces their
 dissimilarity.  By doing this repeatedly, we can try to find the
 correct alignment.
 
+You'll remember our astronaut - Eileen Collins - from chapter 3.
```

Comment:
At any rate I'm happy to punt on this point if the other two (graph conceptual ordering and the sentence about the MSE alignment) are dealt with. If you agree that there is something to deal with. =)